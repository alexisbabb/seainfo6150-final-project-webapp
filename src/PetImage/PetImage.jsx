import React from "react";
import PropTypes from "prop-types";
import styles from "./PetImage.module.css";

const PetImage = (props) => {
  return <img className={styles.image} alt={props.name} src={props.url} />;
};

PetImage.propTypes = {
  name: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired,
};
export default PetImage;