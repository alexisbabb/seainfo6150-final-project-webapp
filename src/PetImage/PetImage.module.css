.image {
    flex: 1;
    width: 100%;  
    border-radius: 10%; 
    object-fit: cover; 

}