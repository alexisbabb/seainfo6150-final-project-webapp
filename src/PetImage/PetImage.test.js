import React from "react";
import PetImage from "./PetImage"

describe("PetImage tests", () => {
    it("renders correctly", () => {
        const url = "https://pbs.twimg.com/media/DlmBBTXXcAEdn6V.jpg";
        const name = "Aladdin";
        const { container } = render(<PetImage url={url} name = {name}/>);
        expect(container).toMatchSnapshot();
    });
});