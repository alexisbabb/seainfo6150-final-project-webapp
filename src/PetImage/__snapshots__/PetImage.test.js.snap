// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PetImage tests renders correctly 1`] = `
<div>
  <img
    alt="Aladdin"
    class="image"
    src="https://pbs.twimg.com/media/DlmBBTXXcAEdn6V.jpg"
  />
</div>
`;
